fabric-data-generation-api-v1/src/testmod/java/net/fabricmc/fabric/test/datagen/DataGeneratorTestEntrypoint.java

```diff
@@ -110,6 +124,51 @@ protected void generateRecipes(Consumer<RecipeJsonProvider> exporter) {
 		}
 	}
 
+	private static class ExistingEnglishLangProvider extends FabricLanguageProvider {
+		private ExistingEnglishLangProvider(FabricDataGenerator dataGenerator) {
+			super(dataGenerator);
+		}
+
+		@Override
+		public void generateTranslations(TranslationConsumer translationConsumer) {
+			translationConsumer.add(SIMPLE_BLOCK, "Simple Block");
+			translationConsumer.add(new Identifier(MOD_ID, "identifier_test"), "Identifier Test");
+			translationConsumer.add(EntityType.ALLAY, "Allay");
+			translationConsumer.add(EntityAttributes.GENERIC_ARMOR, "Generic Armor");
+
+			try {
+				Optional<Path> path = dataGenerator.getModContainer().findPath("assets/testmod/lang/en_us.json");
```

Comment:
Isn't this going to cause duplicate assets? I would call that one `en_us.template.json` or something like that.